kafka/conn.py

```diff
@@ -501,10 +501,10 @@ def _send_bytes_blocking(self, data):
                 raise ConnectionError('Buffer overrun during socket send')
             return total_sent
         finally:
-            self._sock.setblocking(False)
+            self._sock.settimeout(0.0)
```

Comment:
You can leave it as setblocking(False). It's the same as per documentation.

Reply:
Yes, they are the same. But I decided to use settimeout here for symmetry with the original call.